aws/modules/deploy/Rakefile

```diff
@@ -118,17 +115,18 @@ task :generate => \
 do
   cp FileList.new("./*.yml"), "#{DEPLOY_OUT_DIR}/"
   begin
-    FileUtils.mkdir "#{DEPLOY_OUT_DIR}/helms"
+    FileUtils.mkdir "#{DEPLOY_OUT_DIR}/charts"
   rescue Errno::EEXIST
   end
-  HELMCHARTS_DIRS.each do |d|
-    FileUtils.cp_r "helms/#{d}", "#{DEPLOY_OUT_DIR}/helms/#{d}"
+  cp "charts/config.yml", "#{DEPLOY_OUT_DIR}/charts/"
+  begin
+    FileUtils.mkdir "#{DEPLOY_OUT_DIR}/charts/values"
```

Comment:
Let's combine this with the above mkdir and elide this rescue block.

Reply:
Done.

Reply:
Heh, actually this is no good because if /charts already exists, /chars/values will not be created :).

Anyway it looks like what we want is `mkdir_p`: https://ruby-doc.org/stdlib-2.4.2/libdoc/fileutils/rdoc/FileUtils.html#method-c-mkdir_p

Also this stanza presumably needs to go before the line that copies config.yml into charts/.

Reply:
Ouch, my head is not working properly today, fixed.